Formula/b/biome.rb

```diff
@@ -0,0 +1,28 @@
+class Biome < Formula
+  desc "Toolchain of the web"
+  homepage "https://biomejs.dev/"
+  url "https://github.com/biomejs/biome/archive/refs/tags/cli/v1.2.2.tar.gz"
+  sha256 "744847d50e716a5ada7eea2f9862fdad6c5321c5aa3ad15c3169e170b2fa09e7"
+  license "MIT"
+  head "https://github.com/biomejs/biome.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(%r{^cli/v(\d+(?:\.\d+)+)$}i)
+  end
+
+  depends_on "rust" => :build
+
+  def install
+    ENV["BIOME_VERSION"] = version.to_s
+    system "cargo", "install", *std_cargo_args(path: "crates/biome_cli")
+  end
+
+  test do
+    (testpath/"test.js").write("const x = 1")
+    system bin/"biome", "format", "--semicolons=always", "--write", testpath/"test.js"
+    assert_match "const x = 1;", (testpath/"test.js").read
+
+    assert_match version.to_s, shell_output("#{bin}/biome --version", 1)
```

Comment:
there is some status code issue with version check, this can be reported to upstream

Reply:
Thanks, I'll look into it. Is this a blocker for merging here ?

Reply:
not a blocker, but just call it out.